packages/main/test/pages/Menu.html

```diff
@@ -110,7 +110,7 @@ <h3 class="header-title">Event logger</h3>
 
 		menu.addEventListener("ui5-before-open", function(event) {
 			const item = event.detail.item;
-			if (item && item.text === "Open" && !fetched) {
+			if (item && item.text.substr(0, 4) === "Open" && !fetched) {
```

Comment:
The change works fine for me, I just think this is redunant.

Reply:
Hi @hinzzx ,

It is not, as I modified little bit this item text in order to be longer.

BR,
Nikolay